dist/src/main/java/io/camunda/zeebe/shared/IdleStrategyConfig.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.shared;
+
+import io.camunda.zeebe.util.EnsureUtil;
+import java.time.Duration;
+import java.util.Objects;
+import java.util.function.Supplier;
+import org.agrona.concurrent.BackoffIdleStrategy;
+import org.agrona.concurrent.IdleStrategy;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration(proxyBeanMethods = false)
+@ConfigurationProperties(prefix = "zeebe.actor.idle")
+public final class IdleStrategyConfig {
+  private static final long DEFAULT_MAX_SPINS = 100;
+  private static final long DEFAULT_MAX_YIELDS = 100;
+  private static final Duration DEFAULT_MAX_PARK_PERIOD = Duration.ofMillis(1);
+  private static final Duration DEFAULT_MIN_PARK_PERIOD = Duration.ofMillis(1);
```

Comment:
🔧 This will now change our defaults, should we maybe test this a bit more?

Reply:
https://github.com/real-logic/agrona/blob/master/agrona/src/main/java/org/agrona/concurrent/BackoffIdleStrategy.java#L93C29-L94C107

Reply:
Before
```
      return new BackoffIdleStrategy(100, 100, 1, TimeUnit.MILLISECONDS.toNanos(1));
```

Reply:
The intention was to keep the defaults, I guess I misread the minParkPeriod (which should be 1ns, not 1ms :s).

Reply:
I don't think anyone has a JVM which can park a thread for 1ns, but why not :smile: